sslutils.proj

```diff
@@ -117,7 +117,7 @@
       <ContribFiles Include="sslutils.sql;uninstall_sslutils.sql"/>
     </ItemGroup>
     <ItemGroup>
-      <ExtensionFiles Include="sslutils--1.3.sql;sslutils--1.2.sql;sslutils--1.1.sql;sslutils--unpackaged--1.0.sql;sslutils--1.0--1.1.sql;sslutils--1.1--1.2.sql;sslutils--1.2--1.3.sql;sslutils--1.0--1.3.sql;sslutils--1.1--1.3.sql;sslutils--unpackaged--1.3.sql;sslutils.control"/>
+      <ExtensionFiles Include="sslutils--1.4.sql;sslutils--1.2.sql;sslutils--1.1.sql;sslutils--unpackaged--1.0.sql;sslutils--1.0--1.1.sql;sslutils--1.1--1.2.sql;sslutils--1.2--1.3.sql;sslutils--1.3--1.4.sql;sslutils--1.0--1.3.sql;sslutils--1.1--1.3.sql;sslutils--unpackaged--1.3.sql;sslutils.control"/>
```

Comment:
We should either keep all version files or remove all.
e.g.
* sslutils--1.3.sql
* sslutils--1.2.sql
* sslutils--1.1.sql